core/src/main/java/hudson/model/Run.java

```diff
@@ -730,9 +730,12 @@ public String getDescription() {
      * Gets the string that says how long the build took to run.
      */
     public @Nonnull String getDurationString() {
-        if(isBuilding())
+        if (hasntStartedYet()) {
```

Comment:
It was a terrible choice of name (not just the misspelled contraction but the use of the negative sense), but too late now.